cishouseholds/pipeline/high_level_transformations.py

```diff
@@ -592,6 +592,120 @@ def transform_survey_responses_version_digital_delta(df: DataFrame) -> DataFrame
         ],
         column_list,
     )
+    df = assign_column_value_from_multiple_column_map(
+        df,
+        "work_status_v1",
+        [
+            [
+                "Employed and currently working",
+                [
+                    "Employed",
+                    "Currently working. This includes if you are on sick or other leave for less than 4 weeks",
+                    None,
+                    None,
+                ],
+            ],
+            [
+                "Employed and currently not working",
+                [
+                    "Employed",
+                    [
+                        "Currently not working -  for example on sick or other leave such as maternity or paternity for longer than 4 weeks",  # noqa: E501
+                        "Or currently not working -  for example on sick or other leave such as maternity or paternity for longer than 4 weeks?",
+                    ],  # noqa: E501
+                    None,
+                    None,
+                ],
+            ],
+            [
+                "Self-employed and currently working",
+                [
+                    "Self-employed",
+                    None,
```

Comment:
should be 
"self-employed"
 "Currently working. This includes if you are on sick or other leave for less than 4 weeks",
None,
None